grazel-gradle-plugin/src/main/kotlin/com/grab/grazel/migrate/android/ResValues.kt

```diff
@@ -17,6 +17,7 @@
 package com.grab.grazel.migrate.android
 
 import com.android.build.gradle.BaseExtension
+import com.android.builder.utils.agpReferenceDocsVersion
```

Comment:
I think this class is not available pre 7.2, is it better to get the AGP version by either plugin or buildscript classpath

Reply:
checked the previous version and yeah this is the case. Changing it to refer to `com.android.builder.model.Version` class instead